app/controllers/concerns/two_factor_authenticatable_methods.rb

```diff
@@ -110,7 +110,7 @@ def handle_valid_otp_for_context(auth_method:)
     if UserSessionContext.authentication_or_reauthentication_context?(context)
       handle_valid_otp_for_authentication_context(auth_method: auth_method)
     elsif UserSessionContext.confirmation_context?(context)
-      handle_valid_otp_for_confirmation_context
+      handle_valid_verification_for_confirmation_context
```

Comment:
Just to clarify my understanding, currently this only happens for phones, but eventually we'd want this method to be called for any MFA in the confirmation context?

Reply:
That’s my plan, yeah